src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -353,8 +371,9 @@ def executeSingleMCPIteration(self, iteration_content):
                                           test_harness.getSasTestHarnessInterface()))
 
     # Step 8: Trigger CPAS and wait until completion.
-    logging.info('Step 8: Triggering CPAS.')
-    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
+    logging.info("Step 8: Triggering CPAS.")
+    cpas = ParallelCpas(self)
+    cpas.start()
```

Comment:
CPAS is still triggered at the same time as previously, this simply allows other work to be completed while it executes.